### PR TITLE
Add Debian 9 Support

### PR DESCRIPTION
**Pull request by liberodark**
Have make more test for sure is work and is working great only on debian 9 all apps is ok

### CODE REVIEW

**Comment by liberodark** [comment]
Have Add Debian 9 support / Fix Emby for Debian 8 and 9 / Add filebot https://www.filebot.net/
Now work only on SSH later try to make web gui for this :D

**Comment by liberodark** [comment]
Have make a branch for filebot is better need test :D

**Comment by 3z3qu13l** [comment]
Had to do a fresh install so I tested it on Debian 9.
Installation went very well

File: setup/quickbox-setup [hunk 1]
```diff
@@ -359,6 +359,7 @@ function _askffmpeg() {
 }
 
 # ask user for vsftpd conf (12)
+apt install -y net-tools
```
Comment (inverse) [inline setup/quickbox-setup]: Whats this for?

**Comment by inverse** [comment]
@liberodark changes theem reasonable - looks like there is a conflict - can you update?

**Comment by liberodark** [comment]
need to review the code.

**Pull request merged**